src/factory.ts

```diff
@@ -82,19 +82,17 @@ function runCommand(
 ): RunResult {
   let result: RunResult;
   const cmdOpts = {command, options};
-  if (isReleasePRCmd(cmdOpts)) {
+  if (isGitHubReleaseCmd(cmdOpts)) {
+    result = factory.call(githubRelease(cmdOpts.options), 'run');
+  } else if (isReleasePRCmd(cmdOpts)) {
     let method: ReleasePRMethod = 'run';
     if (command === 'latest-tag') {
       method = 'latestTag';
     }
     result = factory.call(releasePR(cmdOpts.options), method);
-  } else if (isGitHubReleaseCmd({command, options})) {
-    result = factory.call(githubRelease(cmdOpts.options), 'run');
   } else {
     throw new Error(
-      `Invalid command(${cmdOpts.command}) with options(${JSON.stringify(
-        cmdOpts.options
-      )})`
+      `Invalid command(${command}) with options(${JSON.stringify(options)})`
```

Comment:
I'm not a huge fan of this complex type checking code - when I made `releaseType` optional on both `{GitHubRelease,ReleasePR}FactoryOptions` it introduced a type error in the original code here saying `cmdOpts` had ["acquired the `never` type"](https://www.typescriptlang.org/docs/handbook/basic-types.html#never) after the `isReleasePRCmd` type predicate/guard check. I'm not exactly sure why, perhaps because now `ReleasePRFactoryOptions` is a subset of `GitHubReleaseFactoryOptions` (it used to have a required `releaseType` property). It hurts my brain a little but it works switching the if/else-if conditionals around